openfpga_flow/tasks/fpga_bitstream/repack_ignore_nets/config/task.conf

```diff
@@ -0,0 +1,50 @@
+# = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
```

Comment:
If this task is running fine, please add it to the regression test script:

https://github.com/lnis-uofu/OpenFPGA/blob/9be984f0f3394d00b94b0431bb58d4c3bc374fee/openfpga_flow/regression_test_scripts/fpga_bitstream_reg_test.sh#L31

Currently, we do not support any expected failures in regression tests. So please hide the tests on syntax errors.